Apps/Admin/Server/Controllers/AgentAccessController.cs

```diff
@@ -51,20 +51,21 @@ public AgentAccessController(IAgentAccessService agentAccessService)
         /// <returns>The created agent.</returns>
         /// <param name="agent">The agent model.</param>
         /// <response code="200">Returns the created agent.</response>
-        /// <response code="401">The client must authenticate itself to get the requested response.</response>
+        /// <response code="401">The client must authenticate itself to perform the operation.</response>
         /// <response code="403">
-        /// The client does not have access rights to the content; that is, it is unauthorized, so the server
-        /// is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.
+        /// The client does not have access rights to perform the operation; that is, it is unauthorized.
+        /// Unlike 401, the client's identity is known to the server.
         /// </response>
         /// <response code="502">Unable to get response from Keycloak.</response>
         [HttpPost]
-        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(AdminAgent))]
```

Comment:
This can be made explicit by changing the return type to `Task<ActionResult<AdminAgent>>`